### DISCUSSION TITLE
Kromek RA4S compatiblity

### DISCUSSION THREAD

**Discussion by 2E0WKS** [Open-Gamma-Project/Gamma-MCA#389] [General]
I'm amazed at the MCA and saving spectra from Kromek Kspect as, I didn't have £1000 plus for the Multispect licence and they binned android app years ago. 
There is a github method to extract spe data from device and I see audio setting-but can't seem to enable it.
I'm wondering if Android or raspian would make my setup more mobile?

Thank you for a wonderful MCA

**Comment by NuclearPhoenixx** [comment]
Thanks for the kind words, I'm glad it's useful to you! ❤️ 

Regarding the audio setting, I originally planned to include audio soundcard spectrometry as that can be a major feature that would really complete the MCA. However, it's not trivial and I don't even have a system to test, so I scrapped that for now. Maybe one time I'll get back to this if I want to invest the time and effort.

The great thing about this being a PWA is that you can fully use it on literally any device with a Chromium-based browser (Chrome, Edge, Opera, ...). You just have to open the website in the browser and click the install button. It works in Firefox too, but the serial functionality is not implemented for that browser.
